File: c++/SivitasAkademik.cpp
class SivitasAkademik : public Human {
    private:
        string asalUniv; // atribut asal Universitas
        string emailEdu; // atribut Email edu

    public:
        SivitasAkademik(string nik = "", string nama = "", string jenisKelamin = "", string asalUniv = "", string emailEdu = "") : Human(nik, nama, jenisKelamin) {
            // inisiasi dengan value awal
            this->asalUniv = asalUniv;
            this->emailEdu = emailEdu;
        }

        string getAsalUniv() {
            // return Asal Universitas
            return this->asalUniv;
        }

        string getEmailEdu() {
            // return Email Edu
            return this->emailEdu;
        }

        void setAsalUniv(string asalUniv) {
            // mengganti value Asal Universitas
            this->asalUniv = asalUniv;
        }

        void setEmailEdu(string emailEdu) {
            // mengganti Email Edu
            this->emailEdu = emailEdu;
        }
};

File: python/SivitasAkademik.py
from Human import Human

class SivitasAkademik(Human):
    __asalUniv = ''# atribut asal Universitas
    __emailEdu = ''# atribut Email edu


    def __init__(self, nik = '', nama = '', jenisKelamin = '', asalUniv= '', emailEdu = ''):
      # inisias dengan value awal
      super().__init__(nik, nama, jenisKelamin)
      self.__asalUniv = asalUniv
      self.__emailEdu = emailEdu
    

    def getAsalUniv(self):
      # return Asal Universitas
      return self.__asalUniv
    
    def getEmailEdu(self):
      # return Email Edu
      return self.__emailEdu
    
    def setAsalUniv(self, asalUniv):
      # mengganti value Asal Universitas
      self.__asalUniv = asalUniv
    
    def setEmailEdu(self, emailEdu):
      # mengganti Email Edu
      self.__emailEdu = emailEdu
    

